src/components/IOUConfirmationList.js

```diff
@@ -45,7 +39,7 @@ const propTypes = {
     /** IOU type */
     iouType: PropTypes.string,
 
-    // Selected participants from IOUModal with login
+    /** Selected participants from IOUModal with login */
```

Comment:
Thanks for cleaning up the comments, but not really necessary to do in a PR with so many changes.